microsoft/ReportingServicesTools
# Add support for uploading large PBIX reports

Added check for PBIX report size; If report is large then endpoint for large reports should be used.

Changes proposed in this pull request:
- Added check for PBIX report size
- Added logic for creating a request using endpoint PowerBIReports for loading large reports
- Added getting server configurations related to the maximum possible size of the uploaded file

How to test this code:
- Try to upload a report with a size>100Mb using the Write-RsRestCatalogItem function. Note that this will work, and the rendering speed of the report on the portal will be as fast as if this report were uploaded through the Portal UI 

Has been tested on:
- PowerShell 5
- Windows 10
- PBIRS - 15.0.1107.20
